docker-compose/_posts/2017-07-05-building-a-compose-file-for-runnable.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Building a Compose File for Runnable
+short: Building a Compose File for Runnable
+category: docker-compose
+order: 4
```

Comment:
I wonder if this should be the first doc in the order, since it has to be done before "Setting Up With Compose".